EF6CodeFirstDemo/SchoolContext.cs

```diff
@@ -7,6 +7,7 @@ public class SchoolContext : DbContext
         public SchoolContext() : base("SchoolDB-EF6CodeFirst")
         {
             Database.SetInitializer<SchoolContext>(new SchoolDBInitializer());
+
```

Comment:
Please remove the spaces make it allign.

Reply:
sure ill make changes